### ISSUE TITLE
Integrate momentJS

### ISSUE THREAD

**Issue by artknight** [t1m0n/air-datepicker#100]
This is a very solid and stable lib that works with dates & timezones

**Comment by jiezhang917** [comment]
I agree with this. Currently I have an issue with the timezone, my app is using utc date by default, while air datepicker only accept js date, it's hard to convert a utc rightly to js date as it can only be local timezone for new Date(), does anyone have a solution?

**Comment by pravdinalex** [comment]
+1, maybe just make other version of the component

**Comment by alariva** [comment]
@jiezhang917 I'm facing the [same issue on my application](https://github.com/timegridio/timegrid/commit/7e769ec1c4c12a8b211ce55b676cf3f852201164), will post when I find a workaround.

**Comment by alariva** [comment]
So it looks like I could solve my problem. I think there is no need of integration if you can handle dates with `momentjs` *before talking* to `air-datepicker`.

I took [this reference](http://stackoverflow.com/questions/17987647/moment-js-transform-to-date-object)
